packages/libs/react-ui/src/components/NavAccordion/NavAccordion.css.ts

```diff
@@ -0,0 +1,151 @@
+import { sprinkles } from '@theme/sprinkles.css';
+import { vars } from '@theme/vars.css';
+import { style } from '@vanilla-extract/css';
+
+export const navAccordionWrapperClass = style({});
+
+export const navAccordionListClass = style([
+  sprinkles({
+    margin: 0,
+    padding: 0,
+  }),
+  {
+    listStyleType: 'none',
+    selectors: {
+      '&:last-child': {
+        marginBottom: vars.sizes.$2,
+      },
+    },
+  },
+]);
+
+export const navAccordionListItemClass = style([
+  sprinkles({
+    alignItems: 'center',
+    display: 'flex',
+    lineHeight: '$normal',
+    margin: 0,
+    position: 'relative',
+    padding: 0,
+    paddingLeft: '$5',
+  }),
+  {
+    listStyleType: 'none',
+    selectors: {
+      '&:before': {
+        content: '·',
+        fontSize: vars.fontSizes.$xl,
+        lineHeight: vars.lineHeights.$normal,
+        paddingRight: vars.sizes.$2,
+        verticalAlign: 'middle',
+        position: 'absolute',
+        left: vars.sizes.$1,
+      },
+    },
+  },
+]);
+
+export const navAccordionGroupButtonClass = style([
+  sprinkles({
+    alignItems: 'center',
+    marginBottom: 0,
+    textAlign: 'left',
+  }),
+  {
+    justifyContent: 'flex-start',
+    paddingBottom: '0 !important',
+  },
+]);
+
+export const navAccordionGroupTitleClass = style([
+  sprinkles({
+    fontWeight: '$normal',
+    paddingLeft: '$1',
+  }),
+]);
+
+export const navAccordionGroupIconClass = style([
+  sprinkles({
+    color: '$layoutSurfaceOverlay',
+  }),
+  {
+    transform: 'rotate(-90deg)',
+    transition: 'transform 0.2s ease',
+    selectors: {
+      '&.isOpen': {
+        transform: 'rotate(0deg)',
+      },
+    },
+  },
+]);
+
+export const navAccordionGroupListClass = style([
+  sprinkles({
+    display: 'none',
+    margin: 0,
+    marginTop: '$2',
+    marginBottom: '$2',
+    marginLeft: '$2',
+    overflow: 'hidden',
+    padding: 0,
+  }),
+  {
+    borderLeft: `1px solid ${vars.colors.$layoutSurfaceSubtle}`,
+  },
+]);
+
+export const navAccordionGroupListItemClass = style([
+  {
+    paddingLeft: '0',
+  },
+]);
+export const navAccordionLinkClass = style([
+  sprinkles({
+    color: '$layoutSurfaceOverlay',
+    paddingY: '$1',
+    textDecoration: 'none',
+  }),
+  {
+    selectors: {
+      '&:active, &:hover': {
+        color: `${vars.colors.$negativeSurface} !important`,
+      },
+      '&:hover': {
+        textDecoration: 'underline',
+      },
+      // Top level links
+      'nav > &': {
```

Comment:
nit: I would recommend using the actual className that's used on the nav element if possible to make it more specific